com.unity.build-report-inspector/Editor/Mobile/AndroidUtilities.cs

```diff
@@ -0,0 +1,283 @@
+#if UNITY_ANDROID
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Android;
+using UnityEngine;
+
+namespace Unity.BuildReportInspector.Mobile
+{
+    internal class AndroidUtilities : IPlatformUtilities
+    {
+        private enum ApplicationType
+        {
+            Apk,
+            Aab
+        }
+
+        private static bool IsTestEnvironment
+        {
+            get { return !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("BOKKEN_RESOURCEID")); }
+        }
+
+        private static string JdkPath
+        {
+            get
+            {
+#if UNITY_2018
```

Comment:
I remembered you are going to make this package only compatible with 2019.3 for mobile in yesterday's weekly meeting, correct me if I'm wrong. Then probably we can remove all these ifdef in the future.

Reply:
Fixed